### DISCUSSION TITLE
Create a GitHub Organization to Manage onedarkpro-related Themes

### DISCUSSION THREAD

**Discussion by jtr860830** [olimorris/onedarkpro.nvim#250] [Ideas]
Hello, I’m a big fan of onedarkpro.nvim. I’ve noticed that you’ve also developed themes for other tools like tmux, wezterm, etc. (my dotfiles are highly inspired by you 😄).

I’d like to suggest that you consider creating a GitHub organization, similar to [catppuccin](https://github.com/catppuccin/catppuccin), to manage OnedarkPro and its related themes.

**Comment by olimorris** [comment]
Thanks for your feedback and sorry for not replying sooner. I think the creation of an org would indicate a time commitment that I'm probably not willing to keep 😄 .

As of `v2.0.0`, I've made it much easier to add themes for other apps (see the [extras](https://github.com/olimorris/onedarkpro.nvim/tree/main/extras) folder). I will update `CONTRIBUTING.md` in due course to reflect this but would appreciate any PRs.

**Reply by jtr860830** [reply]
Thanks for the thoughtful reply! No worries about the timing—totally understand how managing these things can become a huge time commitment 😄.

I appreciate the heads-up about the extras folder! It’s great to hear that expanding theme support will be more accessible moving forward. I’ll definitely take a closer look at it, and if I can contribute anything meaningful, I’ll be happy to open a PR.

Looking forward to seeing how OnedarkPro evolves—thanks again for all the hard work you’ve put into it!
